### ISSUE TITLE
Empty form after saving

### ISSUE THREAD

**Issue by palagdan** [kbss-cvut/record-manager-ui#234]
After saving the form, revisiting it shows the fields as empty.
![empty_form](https://github.com/user-attachments/assets/ae4df972-2dae-4069-a981-8aa27b27936e)

**Comment by palagdan** [comment]
![ava_record_form](https://github.com/user-attachments/assets/bb024f5c-d0e9-4624-b468-7b565954f7f9)
Ava deploy does not have this issue

**Comment by blcham** [comment]
@palagdan let's discuss this on next meeting to give you some hints what is wrong

**Comment by VojtechLunak** [comment]
@blcham

**Comment by blcham** [comment]
We cannot fixed this as it we need to wait for akaene ticket that will resolve it.

**Comment by blcham** [comment]
This issue is blocked by another issue.
